### PR TITLE
Feature/1574 year filter

### PR DESCRIPTION
**Pull request by zamuzakki**
This is additional PR for #1574 , to remove Year filter in Trends tab.
Screenshot from 2023-11-10 09-12-06

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 15 sec

File: django_project/frontend/src/containers/MainPage/Trends/index.tsx [hunk 1]
```diff
@@ -79,8 +79,8 @@ const Trends = () => {
                     <PopulationTrend
                       selectedSpecies={selectedSpecies}
                       propertyId={propertyId}
-                      startYear={startYear}
-                      endYear={endYear}
+                      startYear={1960}
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Trends/index.tsx:82]: Why this specific year?

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 50 sec

File: django_project/frontend/src/containers/MainPage/Trends/index.tsx (hunk 1 above)
Comment (zamuzakki) [reply django_project/frontend/src/containers/MainPage/Trends/index.tsx:82]: Because that is the minimum allowed year range start.

**Pull request merged**